--- origsrc/kalgebra-4.7.0/analitzagui/CMakeLists.txt	2011-05-20 15:33:50.000000000 -0500
+++ src/kalgebra-4.7.0/analitzagui/CMakeLists.txt	2011-08-07 03:15:57.017836700 -0500
@@ -41,6 +41,7 @@ if(INSTALL_ANALITZA_HEADERS)
 		algebrahighlighter.h
 		analitzaguiexport.h
 		operatorsmodel.h
+		variablesmodel.h
 
                 DESTINATION ${INCLUDE_INSTALL_DIR}/analitzagui/ COMPONENT Devel)
 endif(INSTALL_ANALITZA_HEADERS)
